assets/sass/patterns/molecules/personalised-cover-download.scss

```diff
@@ -54,27 +66,26 @@
 
 .button-collection {
   display: grid;
-  grid-row: 2;
   justify-items: center;
 }
 
-.button-collection__item {
-  @include margin(12, "left");
-  @include margin(12, "right");
-}
+ol.personalised-cover-buttons[data-behaviour-initialised="true"] {
```

Comment:
I believe this should be `.personalised-cover-buttons[data-behaviour-initialised="true"]`